test/Project.toml

```diff
@@ -19,6 +19,7 @@ MPIPreferences = "3da0fdf6-3ccc-4f1b-acd9-58baa6c99267"
 NCDatasets = "85f8d34a-cbdd-5861-8df4-14fed0d494ab"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
+PrettyTables = "08abe8d2-0d0c-5749-adfa-8a2ac140af0d"
```

Comment:
Just curious, why did we not need PrettyTables before?

Reply:
Before, it was in our main top-level Project. That worked because `coupled_simulation.jl` and `clock.jl` were in `src/`, and the tests just loaded their modules. Now that they're moved to `experiments/`, I'm loading the files from the tests (e.g. [here](https://github.com/CliMA/ClimaCoupler.jl/pull/468/files#diff-b6c291f3d19e31f4ed167224c95d3c7f39bee2eff5a18bd2813d5ff370b6a935R4)), so we need to add PrettyTables to both the experiments and test environments. It's only used once [here](https://github.com/CliMA/ClimaCoupler.jl/pull/468/files#diff-23d845a9405e9ab3c4883643314f9b5e5b1213b21a4d7a05430857b47bbce7c2R182), so I would be open to removing that line and then removing these dependencies, but I'm not sure if we want to remove that functionality.

Reply:
Ok makes sense!